src/OrchardCore.Modules/OrchardCore.ContentTypes/Controllers/AdminController.cs

```diff
@@ -170,6 +170,11 @@ public async Task<ActionResult> EditPOST(string id, EditTypeViewModel viewModel)
             viewModel.DisplayName = contentTypeDefinition.DisplayName;
             viewModel.Editor = await _contentDefinitionDisplayManager.UpdateTypeEditorAsync(contentTypeDefinition, this);
 
+            if(!IsAlphaNumeric(viewModel.Settings["ContentTypeSettings"].Value<string>("Stereotype")))
```

Comment:
Strange!! why the format is looks different?!!

Reply:
space between `if` and `(`

Reply:
I am just doing it because you are the one who reminds it all the time ;)